cromwell.examples.conf

```diff
@@ -622,12 +622,12 @@ database {
   #
   # This feature should be considered experimental and likely to change in the future.
 
-  metadata {
-    #profile = "slick.jdbc.HsqldbProfile$"
-    #db {
-    #  driver = "org.hsqldb.jdbcDriver"
-    #  url = "jdbc:hsqldb:file:metadata-${uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
-    #  connectionTimeout = 3000
-    #}
-  }
+  #metadata {
+  #  profile = "slick.jdbc.HsqldbProfile$"
+  #  db {
```

Comment:
I thought that recreating the key `metadata` wouldn't erase an existing key `metadata.db`? I thought that was the reason why we couldn't easily remove certain conf entries which exist in the default?

Reply:
In this case, the code is detecting that the stanza `metadata` exists, then failing to find the `metadata.db` stanza.